tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -477,29 +401,47 @@ public function userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles(s
 	 * @throws Exception|GuzzleException
 	 */
 	public function theseUsersHaveBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles(TableNode $table):void {
-		$originalSkeletonPath = $this->setSkeletonDirByType($this->getSmallestSkeletonDirName());
-		try {
-			$this->createTheseUsers(true, true, true, $table);
-		} finally {
-			// restore skeleton directory even if user creation failed
-			$this->setSkeletonDir($originalSkeletonPath);
-		}
+		$this->usersHaveBeenCreated($table);
 	}
 
 	/**
-	 * @Given /^these users have been created without skeleton files ?(and not initialized|):$/
-	 * expects a table of users with the heading
-	 * "|username|password|displayname|email|"
-	 * password, displayname & email are optional
+	 * @Given the administrator has created a new user with the following attributes:
+	 * @Given the user :byUser has created a new user using the Graph API with the following settings:
```

Comment:
we can drop `using the Graph API ` as this is the given step can be a part of next PR as this might be a big change
```suggestion
	 * @Given the user :byUser has created a new user with the following settings:
```